Anature/src/application/controllers/BattleController.java

```diff
@@ -826,12 +838,12 @@ public void updateElements(Player player, Trainer enemyTrainer)
 
 		mAnatureFront.setImage(enemyCurr.getFrontSprite());
 
-		startIntro(player, enemyTrainer, enemyCurr);
+		startInto(player, enemyTrainer, enemyCurr);
 
 		mFightManager = new FightManager(player.getAnatures(), enemyTrainer.getAnatures(), player.getName(), enemyTrainer.getName());
 	}
 
-	private void startIntro(Player player, Trainer enemyTrainer, Anature enemyCurr)
```

Comment:
Change this back.

Reply:
I changed that back. What you see now is that I changed the Types to Interface types. This is so we can hit methods that we don't want full access to. Also to start negotiating what should and should not be on a data object.